src/service/Configurator_6D.tpp

```diff
@@ -100,8 +99,8 @@ void Configurator_6D<SERVICE>::init(CmdArgs const& args) noexcept {
 
 
 	/* apply grid displacement */
-	auto pivot = h.pivots[0];
-	grid->translate(-make_real3(pivot.x,pivot.y,pivot.z));
+//	auto pivot = h.pivots[0];
```

Comment:
Why don't we need the grid shift anymore? if I move the receptor by calling pivotize(), don't we need to shift the grid as well?

Reply:
das skript was ich von dem Konfigurator habe hat zumindest für men Beispiel das grid automatisch geshiftet. Die Mitte des Grids lag dann ohne shift schon automatisch an den koordinaten des pivots

Reply:
aber dieses Skript kann doch nicht wissen, welchen pivot wir für den receptor im dof-file benutzen. in meinen Augen sollte das Grid um den Schwerpunkt (SP) des receptors erzeugt werden, auch wenn der SP nicht mit dem globalen Koordinatensystem (KS) zusammenfällt. Wenn wir den receptor dann auf das globale KS shiften (pivotize()) müssen wir das grid ebenfalls shiften. oder nicht?

Vielleicht war in deinem fall der pivot gleich 0?

Reply:
Ok ich habs mir gerade nochmal angeschaut. Dass der Rezeptor mit dem grid übereinstimmen muss hast du vollkommen recht. Habs mir grad nochmal angeschaut. 
Also der ursprung des Grids liegt auf jeden Fall in der Nähre des Schwerpunkt/Pivotkoordinaten . In dem Fall von 1AVX liegt der Schwerpunkt/autopivot bei {8.959 56.91, 92.525} und der gridursprung bei {-13.463, 33.30, 65.42} also um dx= -22 bis dy=-23 dz=-27 in jede Richtung verschoben. 

Fehlerursprung:
Das alte attract wechselt zwischen ungepivoten und gepivoten Koordinaten hin und her . In dem Fall der Liganden-Kraftberechnung  nimmt attract die ungepivoten Koordinaten des Rezeptor. Das grid war in meinem Fall also in bezug auf die alte Version in der richtigen position und die Rezeptorkoordinaten in Bezug auf die alte Version aber falsch. Das richtige ist wahrscheinlich herausgekommen, da das scoring mit Entfernungen gerechnet hat die außerhalb der NL berechnung lagen.

Reply:
habs gelöst der branch ist hinfällig. Die lösung ist in integration of receptor force

Reply:
kann ich den branch und den pull request also löschen?

Reply:
jo